data/coalition/coalition missions.txt

```diff
@@ -2335,3 +2335,469 @@ mission "Coalition: Submarines 4"
+			`	After Mimodum's voice has disappeared, Sedlitaris urges you to take off. "Heard of this 'anomaly' before, I had not. Look for the system it's in, at once we must. Hopefully still there after the centuries, it is."`
+				accept
+	to complete
+		has "Coalition: Expeditions Past 7B: failed"
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`
+
+mission "Coalition: Expeditions Past 7B"
+	invisible
+	landing
+	to offer
+		has "Coalition: Expeditions Past 6: done"
+	on enter "Terminus"
+		conversation
+			`Sedlitaris fiddles with the pylon you recovered from <origin>, and lets the second recorded log play while she gazes out to see the "crimson anomaly" Mimodum had mentioned.`
+			`	"Tried just about everything now, we have. A finisher barrage, bombardment from afar, driving asteroids through it, even activating our jump drive while immersed in the red cloud. Told me that it shows some signs of rotation, the chief researcher has, tried to show me the axis around which it spins, she did. Whatever the case, find anything worthwhile about the cloud, we could not. Returning to Summit now, we are, and from there to regroup with the fleet."`
+			`	The recording ends, but Sedlitaris continues looking out to the "cloud" for a few minutes, in silence.`
+			choice
+				`	"Are you going to play the last one?"`
+					goto last
+				`	"Is something wrong?"`
+			`	"No, just... taking in the view, I was. Unlike any phenomena from within the Coalition, this is."`
+			label last
+			`	She moves back to the pylon, and hesitates for a moment. "Back to Summit, Mimodum said they would go. Better understand his next steps, we would, if listen to the last entry on that clearing where we found this, we did," she says, tapping the pylon.`
+			`	You tell her you'll go back to <origin> soon, and she heads to her bunk.`
+		fail
+	to complete
+		never
+
+mission "Coalition: Expeditions Past 8"
+	landing
+	name "Lost Checkpoint"
+	description "Travel to the marked systems so that Sedlitaris may look for a signal frequency of the expedition."
+	passengers 1
+	to offer
+		has "Coalition: Expeditions Past 7: done"
+	source "Watcher"
+	waypoint "Rasalhague"
+	waypoint "Cebalrai"
+	waypoint "Zeta Aquilae"
+	waypoint "Ascella"
+	waypoint "Peacock"
+	waypoint "Kaus Australis"
+	on offer
+		conversation
+			`You land next to the forest again, and venture into it once more. The trail fresh in your memory, you two reach the clearing quickly, where Sedlitaris puts the pylon close to where it was before being dug out. "Almost like if we were there with them, this should be, being right where they left the pylon. Hopefully a better understanding of what they were thinking, it will also provide."`
```

Comment:
```suggestion
			`You land next to the forest again and venture into it once more. The trail fresh in your memory, you two reach the clearing quickly, where Sedlitaris puts the pylon close to where it was before being dug out. "Almost like if we were there with them, this should be, being right where they left the pylon. Hopefully a better understanding of what they were thinking, it will also provide."`
```